mods/boats/init.lua

```diff
@@ -172,7 +172,7 @@ function boat.on_step(self, dtime)
 		end
 	end
 	local velo = self.object:get_velocity()
-	if self.v == 0 and velo.x == 0 and velo.y == 0 and velo.z == 0 then
+	if not self.driver and self.v == 0 and velo.x == 0 and velo.y == 0 and velo.z == 0 then
```

Comment:
```
	if not self.driver and
			self.v == 0 and velo.x == 0 and velo.y == 0 and velo.z == 0 then
```

Reply:
I agree, as otherwise this goes beyond 90 characters (with tab size 4).

Reply:
Done.